libraries/AP_RCTelemetry/AP_VideoTX.cpp

```diff
@@ -61,9 +61,15 @@ const AP_Param::GroupInfo AP_VideoTX::var_info[] = {
     // @DisplayName: Video Transmitter Options
     // @Description: Video Transmitter Options.
     // @User: Advanced
-    // @Bitmask: 0:Pitmode
+    // @Bitmask: 0:Pitmode,1:Unlocked
```

Comment:
add in description "Do not enable the Unlocked option unless you have appropriate permissions in your jurisdiction to transmit at high power levels"
It should also describe what Pitmode is
pitmode on disarm sounds nice

Reply:
having pitmode on disarm or just lower power setting is indeed nice, but we should only be sure it goes there only when we power up the first time and not on next disarms (you crash and disarm you want full video power to be able to find your machine for instance), also we need to be sure that power doesn't change in case we watchdog reset since the last thing you want when flying is to loose the control due to wdt and the video from the machine even briefly.

Reply:
I'll second this, though I'd point out that it's better if this is an option, as many people fly in groups and would prefer to have pit mode on all disarms, to not disturb other fliers when they land.

Reply:
I added both options